projects/demo/src/app/app.component.ts

```diff
@@ -1,9 +1,22 @@
 import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {WebWorker, WorkerExecutor} from '@ng-web-apis/workers';
 
 @Component({
     selector: 'main',
     templateUrl: './app.component.html',
     styleUrls: ['./app.component.less'],
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class AppComponent {}
+export class AppComponent {
+    public workerThread: WebWorker<string, string>;
```

Comment:
nit: we do not use the "public" word cause it is "public" by default

I think we need to add it into linters, hmm